src/libstd/build.rs

```diff
@@ -79,67 +67,3 @@ fn main() {
         println!("cargo:rustc-link-lib=compiler_rt");
     }
 }
-
-fn build_libbacktrace(target: &str) -> Result<(), ()> {
-    let native = native_lib_boilerplate(
```

Comment:
At least one drawback of this change is that `backtrace-sys` is no longer built once for all stages, but is built again for each stage.

Reply:
True! Nowadays though it uses the `cc` crate instead of a configure script, so it's much faster. It's only 4-5 object files ish